metric/push/no_cache/average

```diff
@@ -1,15 +1,17 @@
 # 10 people, 45000 messages total
 # Operation, Average Time (ms)
-# Commit hash: 15f2596f363566975b6f0540d92f7871191c1b94
+# Commit hash: 305817756542bfb2ace6213431df5d6dd8c685cf
```

Comment:
How come the commit change?
We are supposed to use the same commit as last time right?